### PR TITLE
FE-1411 inline inputs

### PR DESCRIPTION
**Pull request by grabkowski**
Updated the InlineInputs component to match new DLS requirements

### CODE REVIEW

File: src/components/inline-inputs/inline-inputs.stories.js [hunk 1]
```diff
@@ -1,25 +1,64 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { text } from '@storybook/addon-knobs';
-import InlineInputs from './inline-inputs';
-import Textbox from '../textbox';
+import { State, Store } from '@sambego/storybook-state';
+import InlineInputs from './inline-inputs.component';
```
Comment (izakrzewska) [inline src/components/inline-inputs/inline-inputs.stories.js]: `import InlineInputs from '.';`?

Comment (grabkowski) [reply src/components/inline-inputs/inline-inputs.stories.js]: done

File: src/components/inline-inputs/inline-inputs.component.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  /** The id of the corresponding input control for the label */
+  htmlFor: PropTypes.string
+};
+
+InlineInputs.defaultProps = {
+  children: null,
+  className: ''
+};
+
+/* eslint-enable react/prop-types */
```
Comment (edleeks87) [inline src/components/inline-inputs/inline-inputs.component.js]: Why is this line here, I can't see where the prop-types were disabled?

Comment (grabkowski) [reply src/components/inline-inputs/inline-inputs.component.js]: I don't know, this file is just renamed (original is `inline-inputs.js`). I haven't toucher that part of code.

Comment (grabkowski) [reply src/components/inline-inputs/inline-inputs.component.js]: removed

**Comment by harpalsingh** [comment]
There is an update we would like to do on the alignment of the inputs.

Currently, we have the inputs sit side by side, but that results in a 2px border:
Screenshot 2019-06-25 at 11 00 45

Could we update to have the borders feel like a single border, therefore each child div, would need a +1 offset (incremental) to make them feel aligned as a single border.

Screenshot 2019-06-25 at 11 00 30

**Pull request merged**